src/actions/index.js

```diff
@@ -0,0 +1,45 @@
+import {connect} from 'react-redux';
```

Comment:
I don't think this file was intended to be included? I don't see it used anywhere...